resources/graphics.py

```diff
@@ -403,6 +403,7 @@
 		'localDispatcherStatusFailed',
 		'localDispatcherStatusKilled',
 		'localDispatcherStatusRunning',
+		'menuBreadCrumb',
```

Comment:
Could you move this and the other menu icons into a `"menus"` group here in this file too please, and add the `"validatePixelAlignment" : True` option, to reflect your nice new reorganisation?

Reply:
Fixed in 70b56893109574a2a199c39ced188678cd8eb278. That also meant changing the `menuIndicator` and `menuIndicatorDisabled` from 8.5px tall to 9px. It doesn't seem to have affected the UI as far as I can tell...